sagemaker-featurestore/feature_store_client_side_encryption.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>leaving the commitment policy default or using '<code>RequireEncryptRequireDecrypt' is best practice for AWS encryption SDK 2.x. Encryption SDK 2.x just came out in september 2020, but I would consider all client side encryption usecases references this document as greenfield.</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>